Source/FieldSolver/FiniteDifferenceSolver/EvolveB.cpp

```diff
@@ -101,16 +109,25 @@ void FiniteDifferenceSolver::EvolveBCartesian (
         amrex::ParallelFor(tbx, tby, tbz,
 
             [=] AMREX_GPU_DEVICE (int i, int j, int k){
+#ifdef AMREX_USE_EB
+                if (Sx(i, j, k) <= 0) return;
```

Comment:
Could you add a comment here:
```suggestion
                // Skip field push if this cell is fully covered by embedded boundaries
                if (Sx(i, j, k) <= 0) return;
```